backend/src/main/java/io/swagger/service/JsonToPojoService.java

```diff
@@ -33,7 +33,10 @@ public class JsonToPojoService {
             e.printStackTrace();
         }
 
-        return new ArrayList<Film>(listOfFilms.stream().map(film -> new Film(film)).toList());
+        return new ArrayList<Film>(listOfFilms.stream().map(film -> {
+            String runningTime = film.getRunningTime().replaceAll("\\D+", "");
```

Comment:
Warum ist muss die runningTime hier nochmal umgeformt werden? Bzw. warum hat film überhaupt ein Feld String runningTime?

Reply:
Wir brauchen ein int kein String. Das problem wäre das Runningtime von Scraper in Form "120 minutes" und wir brauchen nur 120 als int

Reply:
Aber du weist doch der runningTime hier einen String zu oder?

Reply:
ja aber ist automatisch konvertiert und kein Error werfen wie vorher ^^

Reply:
also du kannst 100 minutes zu ein int konvertiert aber 100 kannst du

Reply:
Verstehe ich noch nicht ganz. Wo wird hier ein Error geworfen?